OpenRA.GameMonitor/GameMonitor.cs

```diff
@@ -15,16 +15,44 @@
 using System.Media;
 using System.Reflection;
 using System.Windows.Forms;
+using Microsoft.Win32;
 
 namespace OpenRA
 {
 	class GameMonitor
 	{
+		static readonly string[] VisualStudioRuntimeVersions = { "10.0", "11.0", "12.0" };
+
 		static Process gameProcess;
 
 		[STAThread]
 		static void Main(string[] args)
 		{
+			if (Platform.CurrentPlatform == PlatformType.Windows)
+			{
+				var visualStudioRuntimeInstalled = 0;
+
+				foreach (var version in VisualStudioRuntimeVersions)
+				{
+					var key = Registry.LocalMachine.OpenSubKey(@"SOFTWARE\Microsoft\DevDiv\VC\Servicing\{0}\RuntimeMinimum\".F(version), false);
+					if (key != null && visualStudioRuntimeInstalled == 0)
+						visualStudioRuntimeInstalled = (int)key.GetValue("Install", 0);
+
+					key.Close();
+				}
+
+				if (visualStudioRuntimeInstalled == 0)
+				{
+					var result = MessageBox.Show("OpenRA needs the Microsoft Visual C++ 2010 (or above) x86 Redistributable installed.",
+							"Visual C++ runtime not found.", MessageBoxButtons.OK, MessageBoxIcon.Error);
+
+					if (result == DialogResult.OK)
+						Process.Start("http://www.microsoft.com/en-us/download/details.aspx?id=5555");
```

Comment:
As a user, I really hate it when programs decide they can open random links whenever they feel like it. This is great to help people find what they need, but I'd rather it be optional.
Please replace `MessageBoxButtons.OK` above with `MessageBoxButtons.YesNo` and check if `result == Yes`.
(And add a "Do you want OpenRA to guide you to the download link now?" or something to the message.)